docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,398 @@
+
+You'll notice that when you create a user, you can assign it a role. Don't 
+choose a role yet--we'll come back to that in subsequent steps.  
+
+In <<get-started-elastic-stack>>, you configured {ls} to listen for {metricbeat} 
+input and to send the events to {es}.  You therefore need to create a user 
+that {ls} can use to communicate with {es}. Click *Create user* and create a 
+`logstash_internal` user, for example:
+
+[role="screenshot"]
+image::security/images/create-logstash-user.jpg["Creating a {ls} user in {kib}"]
+
+[float]
+[[gs-roles]]
+=== Assign roles
+
+By default, all users can change their own passwords, get information about 
+themselves, and run the `authenticate` API. If you want them to do more than 
+that, you need to give them one or more _roles_. 
+
+Each role defines a specific set of actions (such as read, create, or delete) 
+that can be performed on specific secured resources (such as indices, aliases, 
+documents, fields, or clusters). To help you get up and running, there are 
+built-in roles. 
+
+Go to the *Management / Security / Roles* page to see them:
+
+[role="screenshot"]
+image::security/images/management-roles.jpg["Role management screenshot in Kibana"]
+
+Select a role to see more information about its privileges. For example, if you 
+select the `kibana_user` role, you will see that it grants `manage`, `read`, 
+`index`, and `delete` privileges on the `.kibana*` indices. To learn more about 
+these privileges, see <<privileges-list-indices>>. 
+
+Let's assign the `kibana_user` role to your user. Go back to the 
+*Management / Security / Users* page and select your user. Add the `kibana_user` 
+role and save the change. For example:
+
+[role="screenshot"]
+image::security/images/assign-role.jpg["Assigning a role to a user in Kibana"]
+
+This user now has the minimum privileges required to use {kib}. 
+
+If you completed all of the steps in <<get-started-elastic-stack>>, you should 
+have {metricbeat} data stored in {es}. Let's create two roles that grant 
+different levels of access to that data. 
+
+Go to the *Management / Security / Roles* page and click *Create role*. 
+
+Create a `metricbeat_reader` role that has `read` and `view_index_metadata` 
+privileges on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-reader-role.jpg["Creating a role in Kibana"]
+
+Create a `metricbeat_writer` role that has `manage_index_templates` and `monitor` 
+cluster privileges, as well as `write`, `delete`, and `create_index` privileges
+on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-writer-role.jpg["Creating another role in Kibana"]
+
+Now go back to the *Management / Security / Users* page and assign these roles 
+to the appropriate users. Assign the `metricbeat_reader` role to your personal 
+user.  Assign the `metricbeat_writer` role to the `logstash_internal` user.
+
+The list of users should now contain all of the built-in users as well as the 
```

Comment:
I really like that you include a screen capture here. It's not good Kibana lets you change tabs without prompting you to Save. You can lose stuff.